scripts/OMakefile

```diff
@@ -127,13 +131,3 @@ install:
 	$(IDATA) gpg/secring.gpg $(DESTDIR)/opt/xensource/gpg/
 	$(IDATA) gpg/pubring.gpg $(DESTDIR)/opt/xensource/gpg/
 	$(IDATA) gpg/trustdb.gpg $(DESTDIR)/opt/xensource/gpg/
-
-sdk-install: install
```

Comment:
Looks like `/etc/rc.d/init.d/sdkinit` was removed. It comes from the work made by Simon